backend/models/survey_post.py

```diff
@@ -81,8 +81,19 @@ def make(post, host):
         """Create personalized json of post."""
         post_dict = super(SurveyPost, post).make(host)
         post_dict["number_votes"] = post.number_votes
-        post_dict["deadline"] = post.deadline.isoformat() if post.deadline else ''
+        post_dict["deadline"] = post.deadline.isoformat(
+        ) if post.deadline else ''
         post_dict["type_survey"] = post.type_survey
-        post_dict["options"] = post.options if post.options else []
+        post_dict["options"] = map(
+            post.make_option, post.options) if post.options else []
 
         return post_dict
+
+    def make_option(post, option):
+        """Create a personalized json of option."""
+        for i in range(len(option["voters"])):
```

Comment:
pq não usar um foreach?

Reply:
Porque precisa substituir o valor no array, fiz usando o for each e mudando a variável e ela não substituiu no array. Uma opção seria usando o elemento e index of, mas como necessitaria do índice de qualquer forma fiz assim.